js/controller.js

```diff
@@ -638,7 +638,8 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
           }
         } else if (this.state.playerState === CONSTANTS.STATE.PLAYING) {
           this.state.screenToShow = CONSTANTS.SCREEN.PLAYING_SCREEN;
-        } else if (this.state.playerState === CONSTANTS.STATE.PAUSE) {
+        } else if (this.state.playerState === CONSTANTS.STATE.PAUSE &&
+          this.state.screenToShow !== CONSTANTS.SCREEN.CLOSEDCAPTION_SCREEN) {
```

Comment:
Should we use CLOSED_CAPTION_SCREEN instead? This will make it consistent with other CLOSED_CAPTION constants.

Reply:
\+